.circleci/config.yml

```diff
@@ -2,39 +2,44 @@ version: 2.1
 orbs:
   codecov: codecov/codecov@3.2.2
   node: circleci/node@5.0.2
+  browser-tools: circleci/browser-tools@1.3.0
 
 jobs:
   test:
     parameters:
       test-type:
         type: string
+      python-version:
+        type: string
     environment:
       NODE_OPTIONS: --max-old-space-size=4096
     docker:
-      - image: circleci/python:3.9.2-browsers
+      - image: cimg/python:<< parameters.python-version >>-browsers
     steps:
       - checkout
       - run: mkdir test-reports
       - restore_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
+          key: deps3-{{ .Branch }}-{{ checksum "requirements.txt" }}-<< parameters.python-version >>
       - run:
           name: Install ffmpeg
           command: |
             sudo apt update
             sudo apt install ffmpeg -y
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
       - run:
           name: Install Python deps in a venv
           command: |
-            python3 -m venv venv
+            python -m venv venv
             . venv/bin/activate
             bash scripts/install_gradio.sh
-            bash scripts/install_test_requirements.sh
+            pip install -r test/requirements-python<< parameters.python-version >>.txt
```

Comment:
Turns out some of our test dependencies do not support python 3.7 in their latest versions.

I created separate dependency files for python 3.7 and 3.9 so that we don't force the 3.9 dependencies to match the 3.7 dependencies.

I also thought it was better to be explicit and have separate dependency files for each python version as opposed to having one that supported both. 

If we only had one file that had to be `pip-compiled` in python 3.7, I worry a dev would add a new dependency in the future, run the `create_test_dependencies` script in 3.9 by mistake and get a set of dependencies that would break the 3.7 build.